internal/databases/mongo/models/backup.go

```diff
@@ -1,27 +1,35 @@
 package models
 
-import "time"
+import (
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
 
 // Backup represents backup sentinel data
+// todo: use `internal.GenericMetadata` as base
```

Comment:
It might be useful to write the implementation for [GenericMetaFetcher/GenericMetaSetter](https://github.com/wal-g/wal-g/blob/3ffb8d522d45242f14ef9e69d6b754f15830e5d2/internal/generic_metadata.go#L53), for example like it has been done [in Greenplum](https://github.com/wal-g/wal-g/blob/master/internal/databases/greenplum/generic_meta_interactor.go) to use the common code for `backup-mark`, backup selectors, etc.

Reply:
I have created new task for me to implement it, but I don't want implement it in this PR, because this functionality was implemented before me and not connected with binary feature directly (it is refactoring and should be in separate pr).